README.md

```diff
@@ -60,6 +60,11 @@ On a rooted Google OnHub, the command needs to be run with the "chronos" user. I
     Password:
     $ curl -Ls git.io/vddgY | bash
 
+Help
+----
+
+_Please check out the [wiki](https://github.com/skycocker/chromebrew/wiki) to find out more information about Chromebrew including helpful tips, resource links and frequently asked questions._ **(Please Do before submmiting a Issue)** 
```

Comment:
You might say _(Please read the [wiki](https://github.com/skycocker/chromebrew/wiki) and check existing [Issues](https://github.com/skycocker/chromebrew/issues) before submitting a new request)_.

Reply:
Kk Also should I remove the comment before @JL2210 Gets mad?